scss/_functions.scss

```diff
@@ -54,7 +54,7 @@
   $g: green($color);
   $b: blue($color);
 
-  $yiq: (($r * 299) + ($g * 587) + ($b * 114)) / 1000;
+  $yiq: ($r * 299 + $g * 587 + $b * 114) / 1000;
```

Comment:
I'd keep the brackets here.

Reply:
Yeah, I wasn't completely sure with this one. Just restored it.